content/intro-to-storybook/react/es/using-addons.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Complementos'
+tocTitle: 'Complementos'
+description: 'Aprende como integrar y utilizar el popular complemento Controls'
+commit: '9b5a6d7'
+---
+
+Storybook tiene un ecosistema sólido de [complementos, o "addons"](https://storybook.js.org/docs/react/configure/storybook-addons) que puedes utilizar para mejorar la experiencia del desarrollador para todos en tu equipo. Puedes ver todo los complementos [aquí](https://storybook.js.org/addons).
+
+Si has estado siguiendo este tutorial, ya encontraste varios complementos y configuraste uno en el capítulo [Testing](/intro-to-storybook/react/es/test/).
+
+Hay complementos para cada posible caso de uso, y demoraría una eternidad en escribir sobre todos. Vamos a integrar uno de los complementos más populares: [Controls](https://storybook.js.org/docs/react/essentials/controls).
+
+## Que es Controls?
```

Comment:
@lmac-1 not much of a blocker but a nice to have for accuracy and consistency. If you could take another pass at the translation and include the `¿` for questions and it's equivalent to exclamation marks, I'd appreciate it.

Reply:
Fixed in commit 665ee7c 🙏🏼